pkgs/by-name/di/dingtalk/package.nix

```diff
@@ -0,0 +1,234 @@
+{
+  fetchurl,
+  stdenv,
+  autoPatchelfHook,
+  makeWrapper,
+  lib,
+  makeDesktopItem,
+  copyDesktopItems,
+  # DingTalk dependencies
+  alsa-lib,
+  apr,
+  aprutil,
+  at-spi2-atk,
+  at-spi2-core,
+  cairo,
+  cups,
+  curl,
+  dbus,
+  e2fsprogs,
+  fontconfig,
+  freetype,
+  fribidi,
+  gdk-pixbuf,
+  glib,
+  gnome2,
+  gnutls,
+  graphite2,
+  gtk3,
+  harfbuzz,
+  icu63,
+  krb5,
+  libdrm,
+  libgcrypt,
+  libGLU,
+  libglvnd,
+  libidn2,
+  libinput,
+  libjpeg,
+  libpng,
+  libpsl,
+  libpulseaudio,
+  libssh2,
+  libthai,
+  libxcrypt-legacy,
+  libxkbcommon,
+  mesa,
+  mtdev,
+  nghttp2,
+  nspr,
+  nss,
+  openldap,
+  openssl_1_1,
```

Comment:
OpenSSL 1.1 will be dropped #269713

Reply:
Thanks for letting me know. Since OpenSSL 1.1 is a hard dependency for DingTalk, I will close this PR.